docs/operation/operation.md

```diff
@@ -921,6 +921,12 @@ The parameters of `-passive-health-check` option are:
 + `max-drop-probabilty=<float more than/equal to 0 and less than/equal to 1>` - the maximum possible probability of unhealthy endpoint being not considered
 while choosing the endpoint for the given request
 
+### Metrics
+
+A set of metrics will be exposed to track passive health checks:
+
+* `routing.endpoint.drop.loadbalancer`: Number of endpoints dropped before load balancing the requests 
```

Comment:
It seems to me such sentence could be misunderstood by external reader: it implicitly states that this is number of **distinct** endpoints dropped. However, this counter is incremented once per endpoint and per request.
So I would tell it is something like 'number of times some endpoint was dropped from endpoints' list before applying load balancer algorithm'.

Reply:
hope the change address your concern